core/frontend/src/tile/LRUTileList.ts

```diff
@@ -323,7 +323,8 @@ export class LRUTileList {
     while (prev && prev !== this._sentinel && this.totalBytesUsed > maxBytes) {
       const tile = prev as Tile;
       prev = tile.next;
-      if (!tile.usageMarker.getIsTileInUse())
+      // prevent freeing if AdditiveRefinementStepChildren are in play, as they depend on the parent tile to draw
+      if (!tile.children || !tile.children.some((child) => (child as RealityTile).isStepChild))
```

Comment:
The effect of this is that we will assume at least one of the step children is currently selected, which is not necessarily the case. Eventually the normal periodic pruning of expired tiles will clean them up, but you'll be holding them in memory for longer than they are needed.

Since I don't think we have an efficient way to check if a tile *is* in the selected partition, I'm ok with this approach for now to get this resolved. But don't do it here with the ugly cast. Make RealityTile override `freeMemory` and put the step-child check in that method.

Reply:
Makes sense, done